<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Параметры</string>
  </property>
  <property name="locale">
   <locale language="Russian" country="Russia"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>553</width>
        <height>309</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Калибровочные данные</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="4">
        <widget class="QDoubleSpinBox" name="headHorizonAngleSpinBox">
         <property name="suffix">
          <string> °</string>
         </property>
         <property name="decimals">
          <number>6</number>
         </property>
         <property name="minimum">
          <double>-100.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Угол горизонта:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="headRadiusLable">
         <property name="text">
          <string>Радиус наконечника:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Длина консоли:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QDoubleSpinBox" name="headRadiusSpinBox">
         <property name="suffix">
          <string> нм</string>
         </property>
         <property name="decimals">
          <number>6</number>
         </property>
         <property name="maximum">
          <double>9999999999.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QDoubleSpinBox" name="tipLengthSpinBox">
         <property name="suffix">
          <string> нм</string>
         </property>
         <property name="decimals">
          <number>6</number>
         </property>
         <property name="maximum">
          <double>999999999999.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="headLengthLable">
         <property name="text">
          <string>Длина наконечника:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QDoubleSpinBox" name="headLengthSpinBox">
         <property name="wrapping">
          <bool>false</bool>
         </property>
         <property name="suffix">
          <string> нм</string>
         </property>
         <property name="decimals">
          <number>6</number>
         </property>
         <property name="maximum">
          <double>9999999999999.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QPushButton" name="calibrationButton">
         <property name="text">
          <string>Калибровка</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>553</width>
        <height>309</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Парметры устройства</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Радиус точки крепления: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="mountPointRadiusSpinBox">
           <property name="suffix">
            <string> мм</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Длина на импульс, ось x:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="lengthByImpulseXSpinBox">
           <property name="suffix">
            <string> нм</string>
           </property>
           <property name="decimals">
            <number>6</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Длина на импульс, угол:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="lengthByImpulseYSpinBox">
           <property name="suffix">
            <string> нм</string>
           </property>
           <property name="decimals">
            <number>6</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Минимальный угол консоли:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="minToolAngleSpinBox">
           <property name="suffix">
            <string> °</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Максимальный угол консоли:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="maxToolAngleSpinBox">
           <property name="suffix">
            <string> °</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>553</width>
        <height>309</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Настройки отображения</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QCheckBox" name="anglesInDoubleCB">
           <property name="text">
            <string>Отображать углы в десятичных градусах</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Осьи в</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="axisUnit">
             <item>
              <property name="text">
               <string>миллиметрах</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>нанометрах</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>микрометрах</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>553</width>
        <height>309</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Сервисные настройки</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="openUsersDialog">
         <property name="text">
          <string>Контроллеры</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
